client/stylesheets/utils/_fonts.import.less

```diff
@@ -1,72 +1,47 @@
 @font-face {
-	font-family: 'Roboto Condensed';
-	src: url('/fonts/robotocondensed-light-webfont.eot');
-	src: url('/fonts/robotocondensed-light-webfont.eot?#iefix') format('embedded-opentype'),
-		 url('/fonts/robotocondensed-light-webfont.woff2') format('woff2'),
-		 url('/fonts/robotocondensed-light-webfont.woff') format('woff'),
-		 url('/fonts/robotocondensed-light-webfont.ttf') format('truetype'),
-		 url('/fonts/robotocondensed-light-webfont.svg#roboto_condensedlight') format('svg');
-	font-weight: 300;
-	font-style: normal;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 300;
+  src: local("Roboto Light"), 
+    local("Roboto-Light"), 
+    url("/fonts/roboto-23d439d23da3eaa8144f02e19043fab2b3f19a33.woff2") format("woff2"), 
```

Comment:
webfont-dl generates sha1-hash for filenames.  I couldn't find a way to generate user friendly names.